src/Catty/PlayerEngine/Sensors/Touch/LastFingerIndexSensor.swift

```diff
@@ -39,10 +40,10 @@
     }
 
     func convertToStandardized(rawValue: Double, for spriteObject: SpriteObject) -> Double {
-        return rawValue
+        return Double(Int(rawValue / 2))
```

Comment:
Currently, when a touch begins the sensor keeps evaluating to "0" until the finger is released from the screen. Please (only) increment the number of touches when a touch begins and deal with this issue in the Touch manager itself.